src/client/containers/authentication/authentication.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import FormComponent from '../formComponent/formComponent';
+import { formFieldsInfo } from '../../helpers/constants';
+import authImage from '../../assets/images/authImage.png';
+import './authentication.css';
+
+class Authentication extends Component {
+    state = {
+      isMember: true,
+    };
+
+    onLinkClick = () => {
+      const { isMember } = this.state;
+      this.setState({ isMember: !isMember });
+    };
+
+    filterFields = (arr, status) => (status ? arr.filter(el => status === el.isMember) : arr);
+
+    render() {
+      const { isMember } = this.state;
+      const fildsToRender = this.filterFields(formFieldsInfo, isMember);
+      const formTitle = isMember ? 'Login' : 'Registration';
+      const message = isMember ? 'Dont have an account? Register!' : 'Already have an account? Login!';
+      return (
+        <div className="authentication">
+          <img type="image/svg+xml" src={authImage} alt="CooLEqualizer" className="authImage" />
+          <div className="form-container">
+            <h1 className="title">{formTitle}</h1>
+            <FormComponent
+              isMember={isMember}
+              fieldsToRender={fildsToRender}
+            />
+            <button type="submit" onClick={this.onLinkClick} className="message">{message}</button>
```

Comment:
There is not submit button. Better to use input in this case.

Reply:
done